config.json

```diff
@@ -903,6 +903,32 @@
 					"vat_code": "Standard"
 				}
 			]
+		},
+		{
+			"name": "elasticsearch v5 small ha (aiven)",
```

Comment:
these names should probably follow the pattern from [ADR025](https://team-manual.cloud.service.gov.uk/architecture_decision_records/ADR025-service-plan-naming-conventions/) prefixed with the service name, and match the marketplace names.

The `(aiven)` is not required, that was added to the compose was as part of the decomissioning of those services.

I think they should probably be:

`elasticsearch small-ha-es5` and `elasticsearch small-ha-es6`

I'm also a bit concerned that we are using `es6` rather than semver-style `6.3` as per the ADR? ... as this might pose problems if we need to offer both versions (not something for this PR, since that's in paas-cf land, but we might want to addresses before we enable the service)

Reply:
I'll update the names.

There is only the option for the latest in either the ES5 or ES6 series from Aiven at point of instance creation.

Reply:
Okay ... I still think the "es" is redundent tho ... it would be like "postrges small-pg9.5" ... I'll create a story to bring up at planning